Dockerfile

```diff
@@ -5,10 +5,15 @@
 # under the terms of the MIT License; see LICENSE file for more details.
 
 # Install base image and its dependencies
-FROM python:3.8-slim-buster
-# hadolint ignore=DL3008, DL3013, DL3015
+FROM ubuntu:20.04
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+COPY requirements.txt /code/
+
+# hadolint ignore=DL3008, DL3013
 RUN apt-get update && \
```

Comment:
`apt-get update -y`  perhaps?  (We use the `-y` option in CI recipes, but not in `Dockerfile`s.)